packages/entities/entities-upstreams-targets/src/components/TargetsList.vue

```diff
@@ -40,6 +40,19 @@
       <template #target="{ rowValue }">
         <span class="target-address">{{ rowValue }}</span>
       </template>
+
+      <template
+        v-if="failoverEnabled"
+        #failover="{ rowValue }"
+      >
+        <KBadge
+          appearance="info"
+          class="upstream-failover"
+        >
+          {{ rowValue ? 'Yes' : 'No' }}
```

Comment:
Was considering if we should adding this into `en.json` or maybe simplify this with `Y` and `N`

Reply:
Maybe we can rename this column to something like `Target type`, and the values can be `Failover target` and `Load balance target`?